tests/test_simple.yml

```diff
@@ -6,25 +6,25 @@
       range:
         limit: 5
   rules:
-    - name:
+    - name: r0
```

Comment:
@bzwei How is this file getting used, is it a dormant file in the repo. It doesn't seem like it was used before otherwise it would have failed validation check with json schema.